manifests/o/opentrack/opentrack/2022.3.2/opentrack.opentrack.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: opentrack.opentrack
+PackageVersion: 2022.3.2

```

```suggestion
PackageVersion: 2022.3.2
AppsAndFeaturesEntries:
  - DisplayVersion: opentrack-2022.3.2
    ProductCode: '{63F53541-A29E-4B53-825A-9B6F876A2BD6}_is1'
```
To match with the DisplayVersion entry written to registry (that you can view in `appwiz.cpl`)